pkg/fixtures/loader.go

```diff
@@ -0,0 +1,90 @@
+package fixtures
+
+import (
+	"context"
+	"fmt"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/kernel"
+	"github.com/applike/gosoline/pkg/mon"
+)
+
+type FixtureLoader struct {
+	kernel.BackgroundModule
+	logger       mon.Logger
+	Writers map[string]FixtureWriter
+	Settings []FixtureLoaderSettings `cfg:"fixture_loader"`
+}
+
+func NewFixtureLoader() *FixtureLoader {
+	return &FixtureLoader{}
+}
+
+func (f *FixtureLoader) Boot(config cfg.Config, logger mon.Logger) error {
+	f.logger = logger
+
+	if !config.IsSet("fixture_loader"){
+		f.logger.Info("fixture loader module not configured")
+		return nil
+	}
+
+	var s []FixtureLoaderSettings
+	config.UnmarshalKey("fixture_loader", &s)
+	f.Settings = s
+
+	f.BootFixtureWriters(config, logger)
+
+	for _, writer := range f.Writers {
+		err := writer.WriteFixtures()
+
+		if err != nil {
+			f.logger.Error(err, "error occurred during fixture loading")
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (f *FixtureLoader) BootFixtureWriters(config cfg.Config, logger mon.Logger) {
+	f.Writers = map[string]FixtureWriter{}
+
+	for _,s := range f.Settings {
+		if !s.Enabled {
+			f.logger.Info(fmt.Sprintf("Fixture Writer for Input %s and Output %s not enabled", s.Input.Encoding, s.Output))
+			continue
+		}
+
+		readerFactory, ok := fixtureReaders[s.Input.Encoding]
+
+		if !ok {
+			f.logger.Warn("No reader implemented for input encoding ", s.Input.Encoding)
+			continue
```

Comment:
I would prefer to throw an error here. 

Fixture loading is something that we only activate for tests or for development, so if we start an app with an unsupported configuration it is better to just fail immediately with an error indicating what went wrong.

I think it is a better development user experience: DEVUX